msg/BoundingBox2D.msg

```diff
@@ -7,7 +7,8 @@
 # The 2D position (in pixels) and orientation of the bounding box center.
 geometry_msgs/Pose2D center
 
-# The size (in pixels) of the bounding box surrounding the object relative
-#   to the pose of its center.
+# The total size (in pixels) of the bounding box surrounding the
+# object. The X and Y axes are relative to the center pose
```

Comment:
What is "including rotation" mean?

Reply:
I don't think that entire second sentence is necessary (and honestly is pretty confusing)

Reply:
I agree, it's a confusing wording.

It was an attempt to clarify that the size is not measured in "global frame" XY coordinates, but in "local frame" XY coordinates. In other words, the sizes given are **not** for an [axis-aligned bounding box](http://myselph.de/gamePhysics/formulas/400px-Aabb.png), but for the rotated bounding box.

How about "The X and Y axes are expressed in the frame defined by the the center field" (pulled directly from #27 discussion)? Or do you still feel it's confusing?

Reply:
I think that you don't need to clarify this point, its pretty clear. What that ticket is commenting on is whether the size is full width or the width compared against the center point (like a radius vs diameter). 

I think you just need the word "total" to clarify that point